File: src/main/java/com/mmg/guess/Game.java
package com.mmg.guess;

import java.util.Random;
import java.util.Scanner;

public class Game {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        Random random = new Random();
        int number = 0;
        int secret = random.nextInt(10) + 1;
        System.out.println(secret + "");
        while (secret != number) {
            System.out.println("請輸入一個數字");
            number = scanner.nextInt();
            if (number > secret) {
                System.out.println("低一點");
            } else if (number < secret) {
                System.out.println("高一點");
            } else {
                System.out.println("猜中了!");
            }
        }
    }
}

File: src/main/kotlin/com/mmg/Game.kt
package com.mmg

import jdk.nashorn.internal.runtime.ScriptingFunctions.readLine
import java.util.*

fun main(args: Array<String>) {
    val secret = Random().nextInt(10)+1
    println(secret)
//    val scanner = Scanner(System.`in`)
    var number:Int = 0
    while (number != secret) {
        print("請輸入數字")
//        number = scanner.nextInt()
        number = readLine()!!.toInt()
        if(number>secret){
            println("小一點")
        }else if(number<secret){
            println("大一點")
        }else{
            print("猜中了!!")
        }
    }
}



